cpu/lm4f120/periph/timer.c

```diff
@@ -32,6 +32,7 @@
 /* guard file in case no timers are defined */
 #if TIMER_NUMOF
 
+#define TIMER_MAX_VALUE   (0xffffffff)
```

Comment:
seems weird to have this here

Reply:
To make it properly would require to adapt the driver and its config completely which is see as an extra PR, that's why I used this *work-around* here.